crates/subspace-farmer/src/single_disk_plot/piece_receiver.rs

```diff
@@ -19,7 +19,7 @@ use tokio::time::{sleep, timeout};
 use tracing::{debug, error, info, trace, warn};
 
 /// Defines initial duration between get_piece calls.
-const GET_PIECE_INITIAL_INTERVAL: Duration = Duration::from_secs(1);
+const GET_PIECE_INITIAL_INTERVAL: Duration = Duration::from_secs(2);
```

Comment:
IMHO it makes sense to increase the neighboring constant as well: GET_PIECE_ARCHIVAL_STORAGE_DELAY